crypto/fipsmodule/bn/bytes.c

```diff
@@ -140,27 +140,42 @@ BIGNUM *BN_le2bn(const uint8_t *in, size_t len, BIGNUM *ret) {
   }
   ret->width = (int)num_words;
 
-  // Make sure the top bytes will be zeroed.
-  ret->d[num_words - 1] = 0;
+  bn_little_endian_to_words(ret->d, ret->width, in, len);
 
+  return ret;
+}
+
+void bn_little_endian_to_words(BN_ULONG *out, size_t out_len, const uint8_t *in, const size_t in_len) {
```

Comment:
can we add some unit tests for this new function?

Reply:
I added a couple of tests to EndianTests to provide better coverage for the new functions.